internal/payloads/account_payload_test.go

```diff
@@ -0,0 +1,26 @@
+package payloads
+
+import (
+	"testing"
+
+	"github.com/RHEnVision/provisioning-backend/internal/models"
+	"github.com/stretchr/testify/assert"
+)
+
+func TestListResponse(t *testing.T) {
+	accounts := []*models.Account{
+		&models.Account{
+			ID:            1,
+			OrgID:         "cool-org-12",
+			AccountNumber: nil,
+		}, &models.Account{
+			ID:            2,
+			OrgID:         "awesome-org-34",
+			AccountNumber: nil,
+		},
+	}
+
+	response := NewAccountListResponse(accounts)
+
+	assert.Equal(t, 2, len(response), "the response should have two elements")
+}
```

Comment:
Are you sure you want to test a func which copies an array? https://github.com/ezr-ondrej/provisioning-backend/blob/d5d89ff2e809e67d7c3a50d7f06cb092003d4c85/internal/payloads/account_payload.go#L29

I mean, I am fine with that, I can come up with many other tasks to do other than that tho :-) For example testing the service that lists all pubkeys (thus this function too).